server/jetstream_cluster.go

```diff
@@ -3085,6 +3098,12 @@ func (js *jetStream) processClusterDeleteStream(sa *streamAssignment, isMember,
 	// Go ahead and delete the stream if we have it and the account here.
 	if acc, _ = s.LookupAccount(sa.Client.serviceAccount()); acc != nil {
 		if mset, _ := acc.lookupStream(sa.Config.Name); mset != nil {
+			// shut down monitor by shutting down raft
```

Comment:
I wonder if we should either make function called kickMonitor() and possibly make its own channel which we create at the beginning on monitorStream() as long a mset != nil and we simple close this channel in kickMonitor?

Reply:
is it possible to have monitoStream starte, exit, be started again?